web/src/routes/norms/norm/[:normGuid]/index.vue

```diff
@@ -0,0 +1,29 @@
+<script lang="ts" setup>
+import { ref, watch } from "vue"
+import Norm from "@/domain/Norm"
+import { getNormByGuid } from "@/services/normsService"
+
+const props = defineProps<{ normGuid: string }>()
+const norm = ref<Norm | undefined>()
+async function loadNormByGuid() {
+  norm.value = (await getNormByGuid(props.normGuid)).data
+}
+watch(() => props.normGuid, loadNormByGuid, { immediate: true })
+</script>
+<template>
+  <div class="flex flex-col gap-16 p-16">
+    <div v-if="norm" class="p-40 w-[50rem]">
+      <h1 class="heading-02-regular mb-[2.75rem]">
```

Comment:
For the future: please try to stick with the default spacing numbers instead of explicit `rem` values. We try to sync these values properly with design and configure them in the Tailwind setup. Such exceptions make it really hard to achieve a homogeneous look.